exasol/analytics/audit/audit_query_handler.py

```diff
@@ -71,20 +82,29 @@ def __init__(
             [ParameterType, ScopeQueryHandlerContext],
             QueryHandler[ParameterType, ResultType],
         ],
+        schema_getter: Callable[[ParameterType], str],
+        table_name_prefix_getter: Callable[[ParameterType], str],
+        additional_columns_provider: Callable[
```

Comment:
The additional columns are static and don't depend on the Parameter, they only depend on the caller of this constructor.

Reply:
Ah ok - see next push.